arrow-core/src/main/kotlin/arrow/core/extensions/tuple2/bitraverse/Tuple2Bitraverse.kt

```diff
@@ -50,14 +43,7 @@ fun <G, A, B, C, D> Kind<Kind<ForTuple2, A>, B>.bitraverse(
   "EXTENSION_SHADOWED_BY_MEMBER",
   "UNUSED_PARAMETER"
 )
-@Deprecated(
-  "@extension kinded projected functions are deprecated",
-  ReplaceWith(
-  "bisequence(arg1)",
-  "arrow.core.bisequence"
-  ),
-  DeprecationLevel.WARNING
-)
+@Deprecated("Applicative typeclasses is deprecated. Use concrete methods on Pair")
```

Comment:
nit: `are` vs `is` or `typeclass` vs `typeclasses`

Reply:
```suggestion
@Deprecated("Applicative typeclass is deprecated. Use concrete methods on Pair")
```